tests/common/genesis_file_util.hpp

```diff
@@ -1,10 +1,64 @@
 #pragma once
 
-/////////
-/// @brief forward declaration, using as a hack to generate a genesis.json file
-/// for testing
-/////////
-namespace graphene { namespace app { namespace detail {
+#include <boost/filesystem.hpp>
+#include <fc/io/json.hpp>
+#include <graphene/chain/genesis_state.hpp>
+
+
+#ifdef _WIN32
```

Comment:
Why not create a new file but moved the code to this file? The code is not related to "genesis_file_util".

Reply:
created node_util.hpp